test/distributed/test_symmetric_memory.py

```diff
@@ -1012,6 +1012,7 @@ def _verify_reduce_scatter_result(self, inp, res):
                 atol=1e-01,
             )
 
+    @skipIfRocm
```

Comment:
Nit picking. To be consistent with other multimem UTs, please use `@requires_multicast_support`